pkg/operator/ceph/cluster/cluster.go

```diff
@@ -124,15 +119,6 @@ func (c *cluster) reconcileCephDaemons(rookImage string, cephVersion cephver.Cep
 		return errors.Wrap(err, "failed to execute post actions after all the ceph monitors started")
 	}
 
-	// If this is an upgrade, notify all the child controllers
-	if c.isUpgrade {
-		logger.Info("upgrade in progress, notifying child CRs")
-		err := c.notifyChildControllerOfUpgrade()
```

Comment:
Remind me how child controllers are notified now of upgrades?

Reply:
On CR update, the manager restarts so all reconciles are triggered again.